src/ex2/pokemonClient.js

```diff
@@ -0,0 +1,30 @@
+const URL = "https://pokeapi.co/api/v2/pokemon";
+
+export class PokemonClient {
+    constructor() {
+        this.pokemonName = [];
```

Comment:
plural is more accurate for array
```suggestion
        this.pokemonNames = [];
```